scripts/build.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
+export VERSION=$(git describe --tags --match 'v*' --abbrev=0 | sed 's/^v//')
+
+sed -i '' "s/^version = .*/version = \"$VERSION\"/" contracts/account-history-contract/Cargo.toml
+sed -i '' "s/^version = .*/version = \"$VERSION\"/" contracts/trade-shield-contract/Cargo.toml
+sed -i '' "s/^version = .*/version = \"$VERSION\"/" contracts/financial-snapshot/Cargo.toml
```

Comment:
@amityadav0 is it possible to only call this in a CI environment (`$CI == true`) ?

Reply:
yes